### DISCUSSION TITLE
Bouncing letters, animations

### DISCUSSION THREAD

**Discussion by Mandar1ni** [monkeytypegame/monkeytype#1420] [Ideas] [answered]
Can you please add some feature like animation for every typed letter to bounce it up, so it can looks like a wave from words when you typing fast, or you can add smooth increasing currently typed letter size then smooth decreasing size when user types new letter, so it also can look like a wave, I wish you understand what I mean, please add this features and another option 'animations' in settings for it, that's could be awesome feature for you site. If I could do this self I was does, but I'm just learning python by now, and doesn't familiar with html/CSS, also I just learning English, so sorry for typos or misspelling in my message) I wish you have a nice day!)

**Comment by unrealapex** [comment]
It's a cool idea but wouldn't it make it hard to read the words as you're typing?

**Reply by Mandar1ni** [reply]
on the contrary, I think it will be even easier to read, since this "wave" will be easier to see with the eye than the caret, so it will be much easier and more pleasant to follow the text, although this is of course just my guess

**Comment by Miodec** [comment, marked as answer]
I don't think a per letter animation is possible because on each keypress the word element is compared to the input and updated as a whole (not just the single letter). This means that each letter's animation in the same word resets on each keypress.

**Reply by Mandar1ni** [reply]
ok i got it, but still thx for the answer
